web/modules/custom/block_module/src/Controller/BlockController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Drupal\block_module\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class to display the customblock on content page.
+ */
+class BlockController extends ControllerBase {
+
+  /**
+   * Stores the current user object.
+   *
+   * @var object
+   */
+  protected $user;
+
+  /**
+   * {@inheritDoc}
+   */
+  public function __construct($user) {
+    $this->user = $user;
+  }
```

Comment:
Not sure what you are trying to do here.

Reply:
Missed to add the datatype of $user variable. Fixed.